src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -439,22 +439,19 @@ Path getLlcPath() throws IOException {
         return answer;
     }
 
-    private static final List<String> resourcesList = Arrays.asList(
+    private static final List<String> RESOURCES_BY_EXTENSION = Arrays.asList(
             "frag", "fxml", "css", "gls", "ttf", "xml",
             "png", "jpg", "jpeg", "gif", "bmp",
             "license", "json");
 
-    private  List<String> getResources() {
-        List<String> resources = new ArrayList<>(resourcesList);
+    private List<String> getResources() {
+        List<String> resources = RESOURCES_BY_EXTENSION.stream()
+                .map(extension -> "-H:IncludeResources=.*\\." + extension + "$")
+                .collect(Collectors.toList());
+
         resources.addAll(projectConfiguration.getResourcesList());
```

Comment:
What happens with the user's list? it won't have `-H:IncludeResources=`. The user should only provide, when needed, a list with file extensions.

Reply:
Good catch, I've added the prefix back in.